src/goto-analyzer/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+# Library
+file(GLOB_RECURSE sources "*.cpp")
+file(GLOB_RECURSE headers "*.h")
+list(REMOVE_ITEM sources
+    ${CMAKE_CURRENT_SOURCE_DIR}/goto_analyzer_main.cpp
+
+    # This doesn't build
+    ${CMAKE_CURRENT_SOURCE_DIR}/xml_binaries/xml_goto_function.cpp
```

Comment:
Please file a PR removing this file.

Reply:
To clarify, are you suggesting removing *all* files which aren't mentioned in makefiles and don't build, or just the two that you've pointed out?

Reply:
I'm all up for removing all files that are never mentioned in Makefiles!

Reply:
OK, the pr is #1321